physics/SFC_Models/Land/Noahmp/noahmpdrv.F90

```diff
@@ -690,11 +1050,6 @@ subroutine noahmpdrv_run                                       &
   errmsg = ''
   errflg = 0
 
-!
-!  --- Just return if external land component is activated for two-way interaction
-!
-  if (cpllnd .and. cpllnd2atm) return
```

Comment:
why are these lines removed?

Reply:
unless by mistake, I am not supposed to touch those. I will look.